rotkehlchen/api/rest.py

```diff
@@ -2114,57 +2115,73 @@ def get_info(self, check_for_updates: bool) -> Response:
     def ping() -> Response:
         return api_response(_wrap_in_ok_result(True), status_code=HTTPStatus.OK)
 
-    def _import_data(
-            self,
-            source: IMPORTABLE_LOCATIONS,
-            filepath: Path,
-            timestamp_format: Optional[str],
-    ) -> Dict[str, Any]:
+    def _do_import_data(self, source: str, filepath: Path, **kwargs: Any) -> Tuple[bool, str]:
+        success, msg = False, ''
         if source == 'cointracking.info':
             success, msg = self.rotkehlchen.data_importer.import_cointracking_csv(
                 filepath=filepath,
-                timestamp_format=timestamp_format,
+                **kwargs,
```

Comment:
Changed to `**kwargs` here since extra parameters passed here become `**kwargs` in csv processing functions

Reply:
Why? This is not a good change. We are losing type checking and open ourselves to dynamic errors of argument passing.

Reply:
Hm... Alright, rolled back